src/Data/List/Relation/Ternary/Appending/Properties.agda

```diff
@@ -0,0 +1,42 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of the generalised view of appending two lists
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.List.Relation.Ternary.Appending.Properties where
+
+open import Data.List.Base using (List; [])
+open import Data.List.Relation.Ternary.Appending
+open import Data.List.Relation.Binary.Pointwise as Pw using (Pointwise; []; _∷_)
+open import Level using (Level)
+open import Relation.Binary using (REL; Rel; Trans)
+open import Relation.Binary.PropositionalEquality using (_≡_; refl)
+
+private
+  variable
+    a b c l r : Level
+    A : Set a
+    B : Set b
+    C : Set c
+    L : REL A C l
+    R : REL A B r
+    as : List A
+    bs : List B
+    cs ds : List C
+
+module _  {e} {E : Rel C e} {L′ : REL A C l} {R′ : REL B C r}
+          (LEL′ : Trans L E L′) (RER′ : Trans R E R′)
+          where
+
+  transitive : Appending L R as bs cs → Pointwise E cs ds → Appending L′ R′ as bs ds
```

Comment:
Hmm I think this name (and `trans` in other files) could be improved? I would expect `transitive` to have a type something like

`Appending L R as bs cs → Appending L R cs ds es→ Appending L R (as ++ bs) ds es`?

This is more like a "respects Pointwise" property. Indeed there are two other properties we could add where the pointwise equality is over the first and second list arguments right?